CHANGES.rst

```diff
@@ -90,6 +90,7 @@ API Changes
 - ``astropy.utils``
 
 - ``astropy.visualization``
+  - Created param labelpad in set_xlabel and set_ylabel functions. It is directly passed to set_axislabel as minpad.
```

Comment:
Can you also mention that additional keyword arguments are now also recognized, such as font settings?